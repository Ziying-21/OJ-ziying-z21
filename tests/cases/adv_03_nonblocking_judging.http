###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 385
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:20 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 385
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:20 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:24:21.046Z","id":1,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 385
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:20 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:24:21.051Z","id":2,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 384
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:20 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Running","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:22 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:23 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:24 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:25 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:26 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:27 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:28 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:29 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:30 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 401
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:31 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Running","time":0}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Running","score":0.0,"state":"Running","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 457
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:32 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":147464},{"id":1,"info":"","memory":0,"result":"Time Limit Exceeded","time":10100444}],"created_time":"2022-09-09T02:24:21.043Z","id":0,"result":"Time Limit Exceeded","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":"2022-09-09T02:24:31.294Z"}
###
# Send request:
GET http://127.0.0.1:12345/jobs/1 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 457
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:32 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":135717},{"id":1,"info":"","memory":0,"result":"Time Limit Exceeded","time":10101129}],"created_time":"2022-09-09T02:24:21.046Z","id":1,"result":"Time Limit Exceeded","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":"2022-09-09T02:24:31.288Z"}
###
# Send request:
GET http://127.0.0.1:12345/jobs/2 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 457
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# content-type: application/json
# access-control-allow-credentials: true
# access-control-expose-headers: content-type
# date: Fri, 09 Sep 2022 02:24:32 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":149049},{"id":1,"info":"","memory":0,"result":"Time Limit Exceeded","time":10101267}],"created_time":"2022-09-09T02:24:21.051Z","id":2,"result":"Time Limit Exceeded","score":0.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); loop {} }","user_id":0},"updated_time":"2022-09-09T02:24:31.312Z"}
