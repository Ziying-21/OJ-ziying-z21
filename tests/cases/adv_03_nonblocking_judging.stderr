[2022-09-09T02:24:20Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:24:20Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:24:21Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000163
[2022-09-09T02:24:21Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 385 "-" "-" 0.000582
[2022-09-09T02:24:21Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 385 "-" "-" 0.000818
[2022-09-09T02:24:21Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 385 "-" "-" 0.002125
[2022-09-09T02:24:21Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 384 "-" "-" 0.000290
[2022-09-09T02:24:22Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000510
[2022-09-09T02:24:23Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000276
[2022-09-09T02:24:24Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000350
[2022-09-09T02:24:25Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000287
[2022-09-09T02:24:26Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000216
[2022-09-09T02:24:27Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000345
[2022-09-09T02:24:28Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000263
[2022-09-09T02:24:29Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000337
[2022-09-09T02:24:30Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000327
[2022-09-09T02:24:31Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 401 "-" "-" 0.000217
[2022-09-09T02:24:32Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 457 "-" "-" 0.000315
[2022-09-09T02:24:32Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/1 HTTP/1.1" 200 457 "-" "-" 0.000104
[2022-09-09T02:24:32Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/2 HTTP/1.1" 200 457 "-" "-" 0.000098
